annotation/tests/test_annotations.py

```diff
@@ -50,6 +73,60 @@ class AnnotationRowTable(Base):
 )
 
 
+ANNOTATION_FILE_1 = File(
+    **{
+        "file_id": 1,
+        "tenant": TEST_TENANT,
+        "job_id": 1,
+        "pages_number": 10,
+    }
+)
+
+ANNOTATION_ANNOTATOR = User(user_id="6ffab2dd-3605-46d4-98a1-2d20011e132d")
+
+ANNOTATION_JOB_1 = Job(
+    **{
+        "job_id": 1,
+        "callback_url": "http://www.test.com/test1",
+        "annotators": [ANNOTATION_ANNOTATOR],
+        "validation_type": ValidationSchema.cross,
+        "files": [ANNOTATION_FILE_1],
+        "is_auto_distribution": False,
+        "categories": CATEGORIES,
+        "deadline": None,
+        "tenant": TEST_TENANT,
+        "job_type": JobTypeEnumSchema.ExtractionWithAnnotationJob,
+    }
+)
+
+ANNOTATION_DOC_CATEGORIES = AnnotatedDoc(
+    revision="20fe52cce6a632c6eb09fdc5b3e1594f926eea69",
+    user=ANNOTATION_ANNOTATOR.user_id,
+    pipeline=None,
+    date=SPECIFIC_DATE_TIME,
+    file_id=ANNOTATION_FILE_1.file_id,
+    job_id=ANNOTATION_JOB_1.job_id,
+    pages={},
+    validated=[1],
+    failed_validation_pages=[],
+    tenant=TEST_TENANT,
+    categories=["foo", "bar"],
+    links_json=[],
+)
+
+
+@pytest.fixture
+def mock_db_query_filter_order_by_all_empty():
+    mock_query = MagicMock()
```

Comment:
I really like the way you've chained the parameters using mocks; it's done very well!